### DISCUSSION TITLE
Input date format

### DISCUSSION THREAD

**Discussion by Joshua-Ryan** [taiga-family/taiga-ui#856] [Q&A] [answered]
While looking into the `tui-input-date` component, I noticed that you run the value of the selected date from the calendar through the `TUI_DATE_MASK`. However since this const's reference is not injected and grabbed internally there is no way to provide another mask at the component level to format it differently. For example using slashes instead of dots to separate the day, month and year. I was wondering if there is any way to accomplish this? I have tried to use the `textMask`, however that only works on values that are fully typed in, anything selected from the calendar reverts to the default format.

**Comment by waterplea** [comment, marked as answer]
Customization of date format is in our plans. We definitely must support US way to write them :)
